scripts/src/public/services/offer_contents.ts

```diff
@@ -153,17 +153,16 @@ export async function sumCorrectQuizAnswers(offerContent: db.OfferContent, form:
 	}
 
 	const quiz: Quiz = JSON.parse(translatedContent || offerContent.content);  // this might fail if not valid json without replaceTemplateVars
-	let amountSum = 0;
+	// let amountSum = 0;
 
-	for (const page of quiz.pages) {
+	const amountSum = quiz.pages.reduce((sum: number, page: QuizPage | SuccessBasedThankYouPage): number => {
 		if (page.type === PageType.TimedFullPageMultiChoice) {
-			const p = (page as QuizPage);
-			const answerIndex = p.question.answers.indexOf(answers[p.question.id]) + 1;
-			if (answerIndex === p.rightAnswer) {
-				amountSum += p.amount;
-			}
-		}
-	}
+			if (answers[page.question.id] === page.rightAnswer) {
+				return sum + page.amount;
+			} else { return sum; }
+		} else { return sum; }
+	}, 0);
```

Comment:
Our style guide is to not have one line blocks, but anyway instead of the two `else` clauses just add a ```return sum;``` on this line, just before the end the function so:
```
			if (answers[page.question.id] === page.rightAnswer) {
				return sum + page.amount;
			}
		} 
		return sum; 
	}, 0);
```

Reply:
maybe we can add it to our tslint configuration?

Reply:
Sure